src/Output/Color.php

```diff
@@ -212,6 +187,10 @@ protected function parseCall(string $name, array $arguments): array
             }
         }
 
+        if (isset(static::$styles[strtolower($name)])) {
```

Comment:
if strtolower is not imported already, should be imported (`use function strtolower;` at top)

Reply:
It's already there